pkg/git/git.go

```diff
@@ -747,3 +748,26 @@ func (r *Repo) Rm(force bool, files ...string) error {
 		NewWithWorkDir(r.Dir(), gitExecutable, args...).
 		RunSilentSuccess()
 }
+
+// AddRemote adds a new remote to the current working tree
+func (r *Repo) AddRemote(name, owner, repo string) error {
```

Comment:
As a follow-up (I can take care of that as well): Adding tests for both new API methods.

Reply:
See https://github.com/kubernetes/release/pull/1120